packages/http/libraries/core/src/http/decorators/Controller.ts

```diff
@@ -2,15 +2,18 @@ import {
   getReflectMetadata,
   setReflectMetadata,
 } from '@inversifyjs/reflect-metadata-utils';
+import { BindingScope, injectable } from 'inversify';
 
 import { controllerMetadataReflectKey } from '../../reflectMetadata/data/controllerMetadataReflectKey';
 import { ControllerMetadata } from '../models/ControllerMetadata';
 import { ControllerOptions } from '../models/ControllerOptions';
 
 export function controller(
   pathOrOptions?: string | ControllerOptions,
+  scope?: BindingScope,
```

Comment:
Does it make sense to be part of  `ControllerOptions`?. I know the implementation won't be as clean as the current one but I think the interface makes more sense that way.

Reply:
Yeah, it makes sense. i'm going to refactor it. No problem with new implementation